Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -1,8 +1,41 @@
 let order = [
-  { itemName: "Hot cakes", quantity: 1, unitPricePence: 232 },
+  { itemName: "Hot cakes", quantity: 1, unitPricePence: 2 },
   { itemName: "Apple Pie", quantity: 2, unitPricePence: 139 },
   { itemName: "Egg McMuffin", quantity: 1, unitPricePence: 280 },
   { itemName: "Sausage McMuffin", quantity: 1, unitPricePence: 300 },
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+console.log("QTY\tITEM\t\t\tTOTAL");
+
+let total = 0;
+let maxItemLength = 0; // Track the longest item name
+
+// First, calculate the longest item name for proper alignment
+order.forEach(({ itemName }) => {
+    if (itemName.length > maxItemLength) {
+        maxItemLength = itemName.length;
+    }
+});
+
+// Now, print the table with proper alignment
+order.forEach(({ quantity, itemName, unitPricePence }) => {
+    total += unitPricePence * quantity;
+
+    // Formatting the unitPricePence to include decimal points
+    const formattedPrice = 
```

Comment:
You have chosen quite complex way to achieve your goal. If your goal is to get some initial value in pence (like 1234) and format it as pound and pence (like 12.34), is there a shorter way to do that?

Reply:
Yes I have used basic arithmetic to perform the conversion.